### ISSUE TITLE
Activate acceptance tests for centos-6, debian-7 and ubuntu-14.04 on Travis CI's Docker

### ISSUE THREAD

**Issue by cardil** [coi-gov-pl/puppet-jboss#2]
There is a bug that prevents running of beaker acceptance tests on Travis CI's Docker installation. It is wide spread bug affecting other projects to.

```
/gemshome/gems/beaker-2.24.0/lib/beaker/hypervisor/docker.rb:86:in `block in provision': undefined method `[]' for nil:NilClass (NoMethodError)
	from /gemshome/gems/beaker-2.24.0/lib/beaker/hypervisor/docker.rb:49:in `each'
```

https://travis-ci.org/camptocamp/puppet-openldap/jobs/78376374

COI/JBoss example logs:

 * CentOS 6: https://travis-ci.org/coi-gov-pl/puppet-jboss/jobs/81770815
 * Debian 7: https://travis-ci.org/coi-gov-pl/puppet-jboss/jobs/81770816
 * Ununtu 14.04: https://travis-ci.org/coi-gov-pl/puppet-jboss/jobs/81770817

The bug is rather new...

Notify: @blaa @therek

**Comment by cardil** [comment]
Required fragment in `.travis.yml` file:
```yaml
- rvm: default
  sudo: required
  services: docker
  env: PUPPET_VERSION="3.8.3" RS_SET="centos-6-x86_64-docker"
  script: bundle exec rake acceptance
- rvm: default
  sudo: required
  services: docker
  env: PUPPET_VERSION="3.8.3" RS_SET="debian-7-x86_64-docker"
  script: bundle exec rake acceptance
- rvm: default
  sudo: required
  services: docker
  env: PUPPET_VERSION="3.8.3" RS_SET="ubuntu-14.04-x86_64-docker"
  script: bundle exec rake acceptance
```

Notify: @coi-gov-pl/jboss-puppet-team

**Comment by cardil** [comment]
Fixed in PR #13
